examples/ucp_hello_world.c

```diff
@@ -314,22 +314,20 @@ static int run_ucx_client(ucp_worker_h ucp_worker)
     }
 
     str = calloc(1, test_string_length);
-    if (str != NULL) {
+    if (str == NULL) {
+        fprintf(stderr, "Memory allocation failed\n");
+        ret = -1;
+    } else {
```

Comment:
`goto err_ep`
and no need `else ` branch

Reply:
done